pinot-query-planner/src/main/java/org/apache/pinot/calcite/rel/hint/PinotHintStrategyTable.java

```diff
@@ -108,4 +110,25 @@ public static String getHintOption(List<RelHint> hintList, String hintName, Stri
     }
     return null;
   }
+
+  /**
+   * Get the first hint that has the specified name.
+   */
+  public static RelHint findFirst(Hintable hintable, String hintName) {
```

Comment:
Return is also `Nullable`.
Can we keep the utils consistent naming and calling convention? I'd naturally put this next to `containsHint()` and call it `getHint()`

Reply:
Good catch